apps/condo/domains/common/components/Pickers/DatePicker.tsx

```diff
@@ -39,7 +39,7 @@ const DefaultDatePicker = generatePicker<Dayjs>(dayjsGenerateConfig)
 const generateDatePickerWithLocale = (Picker: PickerType) => (props) => {
     const { locale } = useContext(ConfigProvider.ConfigContext)
 
-    return <Picker css={PickerStyle} locale={locale.DatePicker} {...props} />
+    return <Picker css={PickerStyle} locale={locale?.DatePicker} {...props} />
```

Comment:
Is it even working solution? 
Because right now it seems like in apps they'll be undefined, and you're just escaping them with `?` mark.
And locale in pickers tells a lot about formatting. 

I guess the root of issues is not suppressing the Picker, but identifying why is it missing at first place

Reply:
locales in apps won't be undefined, but i agree i think the best solution is to use locale from Condo itself, but right now i'm at a loss how to do it